jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/InvoicePaymentResource.java

```diff
@@ -143,6 +150,21 @@ public boolean apply(final InvoicePayment input) {
         return Response.status(Response.Status.OK).entity(result).build();
     }
 
+    @TimedResource
+    @GET
+    @Path("/{invoicePaymentId:" + UUID_PATTERN + "}/" + AUDIT_LOG_WITH_HISTORY)
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Retrieve invoice payment audit logs with history by id", response = AuditLogJson.class, responseContainer = "List")
+    @ApiResponses(value = {@ApiResponse(code = 404, message = "Invoice payment not found")})
+    public Response getInvoiceItemAuditLogsWithHistory(@PathParam("invoicePaymentId") final UUID invoicePaymentId,
```

Comment:
Naming to update.

Reply:
Ack, will fix -- and regen + re-release client